src/sentry/migrations/0678_add_query_sub_id_to_aar_activations.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.0.2 on 2024-03-21 18:46
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
```

Comment:
is this the migration you moved to another PR or a different one? (just want to confirm this code should be in this PR)

Reply:
Yes correct, this is the migration that was moved [here](https://github.com/getsentry/sentry/pull/67483)